doc/design/tether/Exec_FSM_Design_Doc

```diff
@@ -0,0 +1,97 @@
+EXEC PROCESS FINITE STATE MACHINE(does not include cvm process/vic-init thought yet... )
```

Comment:
This has not be replaced with my updated version yet. It is not important for this PR so I will pull this and the other documentation out of it.